.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ executors:
   linux-aarch64:
     machine:
       image: ubuntu-2204:current
-    resource_class: arm.medium
+    resource_class: arm.large
```

Comment:
@mencian Should this be reverted back to `medium` now ?

Reply:
Thanks for keeping an eye out! I've reverted the aarch64 machine back to `medium` in [cfbf481](https://github.com/bioconda/bioconda-recipes/commit/cfbf48111a00cc730060b5869990d7c1e41d42c0).